community/io/src/main/java/org/neo4j/io/pagecache/tracing/AutoCloseablePageCacheTracerEvent.java

```diff
@@ -17,14 +17,14 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.neo4j.io.pagecache.monitoring;
+package org.neo4j.io.pagecache.tracing;
 
 /**
  * Some events are desirable to make AutoClosable, but monitoring is not
  * allowed to introduce new failure modes. Thus, this interface overrides
  * the AutoCloseable#close method to not throw any checked exceptions.
  */
-public interface AutoCloseablePageCacheMonitorEvent extends AutoCloseable
```

Comment:
Perhaps this should be moved to the "tracing framework", i.e. something like `org.neo4j.kernel.monitoring.tracing.CloseableEvent`